source/importing-css-assets-and-javascript/index.html.md.erb

```diff
@@ -7,15 +7,36 @@ weight: 70
 
 ## CSS
 
-### Import all the CSS
+To import the CSS from GOV.UK Frontend, you can either:
+
+- add GOV.UK Frontend's CSS file to your HTML
+- import the CSS into your own Sass file
+
+### Add the CSS file to your HTML
+
+If you decide to add the CSS to your HTML, you can do one of the following:
+
+- set up your routing so requests for the CSS file are served from `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css`
+- copy the `node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.css` file into your application
+
+Then link the CSS file before the closing `</head>` tag of your HTML page or page template.
```

Comment:
**suggestion** There's less of a clear point of insertion for CSS as there would be with JS, where right before `<body>` makes it download and run last as you'd want. For CSS, it's more the opposite and you want it to load ASAP so it doesn't block the rendering, so as high up in the `<head>` as possible... but [not necessarily all at the top](https://speakerdeck.com/csswizardry/get-your-head-straight?slide=39). So I'd propose something less specific with:

```suggestion
Then link the CSS file inside the `<head>` tag of your HTML page or page template.
```

Reply:
Cheers, nice way to remove any edge cases. Done ✅